app/models/checkout.rb

```diff
@@ -10,4 +11,11 @@ class Checkout < ApplicationRecord
   def due_date
     self.created_at + CHECKOUT_PERIOD_IN_DAYS.days
   end
+
+  def fine
+    days_overdue = (Time.zone.now - self.due_date).to_i/1.day
+    if days_overdue > 0
```

Comment:
What should the return value of `fine` be for a book that is not overdue? Right now it is `nil`, which means the method returns two types: `nil` or a `Float`. This could lead to a situation down the road where you end up calling a method on `nil` because you expected to always have a number here. Would it be bad to return `0` if the book is not overdue?